lib/features/guide/widgets/about_us/repository/about_us_repository.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+import '../../../../../api_base/watch_query_adapter.dart';
+import '../../../../../config/ttl_config.dart';
+import 'getAboutUsDetails.graphql.dart';
+
+part 'about_us_repository.g.dart';
+
+typedef _GetAboutUs = WatchOptions$Query$getAbousUsDetails;
+typedef AboutUs = Query$getAbousUsDetails$AboutUs;
+typedef AboutUsTeam = Query$getAbousUsDetails$AboutUs_Team;
+
+@riverpod
+Stream<AboutUsDetails?> aboutUsRepository(AboutUsRepositoryRef ref) async* {
+  final stream = ref.watchQueryWithCache(
+    _GetAboutUs(
+      eagerlyFetchResults: true,
+    ),
+    TtlKey.aboutUsRepository,
+  );
+  yield* stream.map((event) {
+    if (event == null) return null;
+    return AboutUsDetails(
+      aboutUs: event.AboutUs,
+      aboutUsTeam: event.AboutUs_Team,
+    );
+  });
+}
+
+
+class AboutUsDetails {
```

Comment:
1. if you want custom model, it shoudl probably go `/model` folder
2. it's best to use `freezed` package for such models
3.  Do you really need this here? it basically works the same as the generated one